app/models/pro/event.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+module Pro
+  ##
+  # Events from Pro JSON API.
+  class Event < Base
+  end
```

Comment:
cool, I hadn't realised this was all in place already, but yes it's all working. There's going to probably be some refactor for when we get the design all implemented, but this all looks good.